monGARS/mlops/model.py

```
@@ -176,3 +191,115 @@ def detach_sequences(sequences: Iterable[Any]) -> list[Any]:
+        )
+    except TypeError:
+        try:
+            model = AutoModelForCausalLM.from_pretrained(
+                model_id, **cpu_kwargs, dtype=cpu_dtype
+            )
+        except TypeError:
+            model = AutoModelForCausalLM.from_pretrained(model_id, **cpu_kwargs)
+
+    try:  # pragma: no cover - defensive cleanup for limited builds
+        model.to("cpu")
+    except Exception:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)